clients/go/cipher.go

```diff
@@ -213,35 +213,38 @@ func (c X25519DeoxysIICipher) DecryptCallResult(response []byte) ([]byte, error)
 		return nil, ErrCallFailed
 	}
 
-	if callResult.Unknown != nil {
-		decrypted, err := c.Decrypt(callResult.Unknown.Nonce, callResult.Unknown.Data)
-		if err != nil {
-			return nil, err
+	var aeadEnvelope AeadEnvelope
+	if callResult.OK != nil {
+		if err := cbor.Unmarshal(callResult.OK, &aeadEnvelope); err != nil {
+			return callResult.OK, nil
 		}
+	} else if callResult.Unknown != nil {
```

Comment:
I thought this path wouldn't occur post `0.5.0`?

Reply:
The client should still work for previous versions.